cdap-client/pom.xml

```diff
@@ -112,6 +112,10 @@
       <artifactId>cdap-explore-client</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
it always did. ``ProgramRunId`` always had a constructor that accepted twill's ``RunId``. We're seeing the dependency here now because previously the ``cdap-client`` module never referenced ``ProgramRunId``.